docker/build.cmd

```diff
@@ -1,2 +1,7 @@
 @echo off
-docker build -t sshnet -f DockerFile .
\ No newline at end of file
+
+rem Build new image
+docker build -t sshnet -f DockerFile .
+
+rem Remove danging images
+docker image prune -f
```

Comment:
I have mixed feelings about this, some people might not like if we automatically prune images they might reuse later, depending on their workflow.

Reply:
Do you know if we can do something to avoid these dangling (<none>) images?
I actually think pruning dangling images is safe, but I could be wrong of course.

Reply:
I don't think we can do something about it, other than prune them. The concern I have is that theoretically user could have some useful data in some of the images. Not very likely, but still...
Also, the sshnet image is less than 25MB, so I don't think it would be an issue if we leave periodic pruning to the user.